.yw_pipeline/yw_pipeline.py

```diff
@@ -52,8 +52,10 @@ async def get(self, _env: YouwolEnvironment, context: Context):
 
 folder_path = Path(__file__).parent.parent
 pkg_json = parse_json(folder_path / "package.json")
+asset_id = encode_id(pkg_json['name'])
+version = pkg_json['version']
 
-assets_dir = f"/api/assets-gateway/raw/package/QHlvdXdvbC9qcy1wbGF5Z3JvdW5k/{pkg_json['version']}/assets"
+assets_dir = f"/api/assets-gateway/raw/package/{asset_id}/{version}/assets"
```

Comment:
The URL needs to be updated to not use this deprecated route:
f"/api/assets-gateway/cdn-backend/resources/{asset_id}/{version}/assets"